### PR TITLE
Fixed #32460 -- Allowed "label"/"do_not_call_in_templates" members in model choice enums.

### PR DESCRIPTION
**Pull request by pope1ni**
An alternate solution to #14020, fixing ticket-32460. See my [comment](https://code.djangoproject.com/ticket/32460#comment:2).

### CODE REVIEW

File: tests/model_enums/tests.py [hunk 1]
```diff
@@ -155,6 +156,23 @@ def test_templates(self):
         output = template.render(Context({'Suit': Suit}))
         self.assertEqual(output, 'Diamond|1')
 
+    def test_special_attributes_and_properties_can_be_members(self):
```
Comment (mkrtchian) [inline tests/model_enums/tests.py:159]: As there are two blocks with comments in this function, wouldn't it be better to make two test functions and remove comments ?

Comment (pope1ni) [reply tests/model_enums/tests.py:159]: Maybe, but in essence they're tests to fix exactly the same issue (although the fix is slightly different for each).

I'll leave it to one of the fellows to decide when they review.

**Comment by felixxm** [comment]
@pope1ni Thanks :+1: I pushed minor edits.

@shaib Do you want to take a look?

**Comment by felixxm** [comment]
Test failure is not related with this patch.

**Comment by shaib** [comment]
Sorry I could not get to this sooner; for what it's worth, I approve.

**Pull request merged**